core/authenticate/authenticate.go

```diff
@@ -18,36 +18,36 @@ func (m AuthMethod) String() string {
 
 // Flow is a temporary state used to finish login/registration flows
 type Flow struct {
-	ID uuid.UUID
+	ID uuid.UUID `db:"id"`
```

Comment:
Although we can use these structs, normally in our postgres implementation we have created a database-specific struct schema for each entity and then they get translated to app-level domain models. This is done to keep them loosely coupled. Check repositories of group or project for reference.